services/brig/src/Brig/Provider/API.hs

```diff
@@ -817,6 +860,36 @@ validateServiceKey pem = liftIO $ readPublicKey >>= \pk ->
         (const $ return Nothing)
         (SSL.readPublicKey (LC8.unpack (toByteString pem)) >>= return . Just)
 
+-- | Send all team members an event about a change in the services
+-- whitelist.
+notifyServiceWhitelistUpdate
```

Comment:
Talked with @neongreen offline and we agreed to postpone the fan out of such events as they are expensive and not strictly necessary at the moment.